SonarQube.Bootstrapper/Resources.resx

```diff
@@ -135,6 +135,9 @@
   <data name="ERROR_FailedToUpdateRunnerBinaries" xml:space="preserve">
     <value>Failed to update the SonarQube MSBuild Runner binaries. Check the server url, verify that the C# plugin is correctly installed on the SonarQube server and that the SonarQube server has been restarted.</value>
   </data>
+  <data name="ERROR_PostProcessExeNotFound" xml:space="preserve">
+    <value>Post-processor executable not found '{0}'. Please check that pre-processing was executed in the current directory.</value>
```

Comment:
@HSAR the "pre" and "post" steps are more technical terms present in the codebase than user-oriented ones

The user is expected to run:
* MSBuild.SonarQube.Runner.exe begin ...
* MSBuild.SonarQube.Runner.exe end

So I'd rather see "begin" and "end" mentioned in the error message.